libpromises/evalfunction.c

```diff
@@ -1300,27 +1300,45 @@ static FnCallResult FnCallBundlesMatching(EvalContext *ctx, const Policy *policy
 
 /*********************************************************************/
 
-static FnCallResult FnCallPackagesMatching(ARG_UNUSED EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
+static bool AddPackagesmatchingJsonLine(pcre *matcher, JsonElement *json, char *line)
```

Comment:
Minor: For consistency I would do `s/AddPackagesmatchingJsonLine/AddPackagesMatchingJsonLine/`.